src/components/UserProfile/index.jsx

```diff
@@ -65,50 +66,52 @@ class UserProfile extends React.Component {
                     />
                   )
                 })
-                : <Cards.ApplyForAVoyageCard />
+                : editable
+                  ? <Cards.ApplyForAVoyageCard />
+                  : (
+                    <div className="no-data-card">
+                      Nothing Here Yet! Check Back Later!
+                    </div>
+                  )
               }
             </section>
             <section className="user-voyage">
               {
                 pendingApproval.length > 0
-                  ? pendingApproval.map((cohort, index) => {
-                    return (
-                      <React.Fragment key={cohort.id + "_" + index}>
-                        <div className="user-voyage-title">Upcoming Voyages</div>
-                        <Cards.PendingApprovalVoyageCard
-                          key={cohort.id + "_" + index}
-                          voyageNumber={cohort.id}
-                          startDate={cohort.start_date}
-                          endDate={cohort.end_date}
-                          cohort={cohort.title}
-                        />
-                      </React.Fragment>
-
-                    )
-                  })
-                  : null
+                && pendingApproval.map((cohort, index) => {
+                  return (
+                    <React.Fragment key={cohort.id + "_" + index}>
+                      <div className="user-voyage-title">Upcoming Voyages</div>
+                      <Cards.PendingApprovalVoyageCard
```

Comment:
how complex would it be to render "Pending Approval" but then add a "APPLY" button if `editable` is true? meaning if you view someone elses profile and they are pending approval you can apply to the same voyage